mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,219 @@
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: metadataExports
+            output: true
+            description: |
+              The latest metadata exports of the metastore service.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Time
+                  name: 'startTime'
+                  description: |
+                    The time when the export started..
+                  output: true
+                - !ruby/object:Api::Type::Time
+                    name: 'endTime'
+                    description: |
+                      The time when the export ended.
+                    output: true
+                - !ruby/object:Api::Type::String
+                  name: 'state'
+                  description: |
+                    The current state of the export.
+                  output: true
+                - !ruby/object:Api::Type::String
+                  name: 'destinationGcsUri'
+                  description: |
+                     A Cloud Storage URI of a folder that metadata are exported to, in the form of `gs://<bucket_name>/<path_inside_bucket>/<export_folder>`.
+                  output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'hiveMetastoreConfig'
+        description: |
+         Configuration information specific to running Hive metastore software as the metastore service.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'version'
+            input: true
+            required: true
+            description: |
+              The Hive metastore schema version.
+          - !ruby/object:Api::Type::KeyValuePairs
+            name: 'configOverrides'
+            description: |
+              A mapping of Hive metastore configuration key-value pairs to apply to the Hive metastore (configured in hive-site.xml).
+              The mappings override system defaults (some keys cannot be overridden)
+          - !ruby/object:Api::Type::NestedObject
+            name: 'kerberosConfig'
+            description: |
+              Information used to configure the Hive metastore service as a service principal in a Kerberos realm.
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'keytab'
+                description: |
+                  A Kerberos keytab file that can be used to authenticate a service principal with a Kerberos Key Distribution Center (KDC).
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'cloudSecret'
+                    description: |
+                      The relative resource name of a Secret Manager secret version, in the following form:
+
+                      "projects/{projectNumber}/secrets/{secret_id}/versions/{version_id}".
+              - !ruby/object:Api::Type::String
+                name: 'principal'
```

Comment:
Would these fields be required as well if the user is specifying the kerberos config?